src/locale/lang/kz.js

```diff
@@ -0,0 +1,109 @@
+export default {
+  el: {
+    colorpicker: {
+      confirm: 'Қабылдау',
+      clear: 'Тазалау'
+    },
+    datepicker: {
+      now: 'Қазір',
+      today: 'Бүгін',
+      cancel: 'Болдырмау',
+      clear: 'Тазалау',
+      confirm: 'Қабылдау',
+      selectDate: 'Күнді таңдаңыз',
+      selectTime: 'Сағатты таңдаңыз',
+      startDate: 'Басталу күні',
+      startTime: 'Басталу сағаты',
+      endDate: 'Аяқталу күні',
+      endTime: 'Аяқталу сағаты',
+      prevYear: 'Алдыңғы жыл',
+      nextYear: 'Келесі жыл',
+      prevMonth: 'Алдыңғы ай',
+      nextMonth: 'Келесі ай',
+      year: 'Жыл',
+      month1: 'Қаңтар',
+      month2: 'Ақпан',
+      month3: 'Наурыз',
+      month4: 'Сәуір',
+      month5: 'Мамыр',
+      month6: 'Маусым',
+      month7: 'Шілде',
+      month8: 'Тамыз',
+      month9: 'Қыркүйек',
+      month10: 'Қазан',
+      month11: 'Қараша',
+      month12: 'Желтоқсан',
+       week: 'Апта',
```

Comment:
The CI fails because of this indentation.

Reply:
okay thanks removed this space